homeassistant/components/tesla_fleet/strings.json

```diff
@@ -412,6 +412,19 @@
       "vehicle_state_odometer": {
         "name": "Odometer"
       },
+      "island_status": {
+        "name": "Grid Status",
+        "state": {
+          "island_status_unknown": "Unknown",
+          "on_grid": "Connected",
+          "off_grid": "Disconnected",
+          "off_grid_unintentional": "Disconnected unintentionally",
+          "off_grid_intentional": "Disconnected intentionally"
+        }
+      },
+      "storm_mode_active": {
+        "name": "Storm Watch active"
+      },
```

Comment:
I storm mode active still in this PR? It was going to be a binary right? If its not in this PR you need to remove it from strings.json